packages/client/modules/teamDashboard/components/TeamArchive/TeamArchive.tsx

```diff
@@ -278,79 +303,98 @@ const TeamArchive = (props: Props) => {
     )
   }
 
-  const _onSectionRendered = ({columnStartIndex, columnStopIndex, rowStartIndex, rowStopIndex}) => {
+  const onSectionRendered = ({columnStartIndex, columnStopIndex, rowStartIndex, rowStopIndex}) => {
     if (!_onRowsRenderedRef.current) return
     _onRowsRenderedRef.current({
       startIndex: getIndex(columnStartIndex, rowStartIndex),
       stopIndex: getIndex(columnStopIndex, rowStopIndex)
     })
   }
 
+  const [isScrolledToBottom, setIsScrolledToBottom] = useState(false)
+  const onScroll = ({clientHeight, scrollHeight, scrollTop}: ScrollParams) => {
+    if (!isScrolledToBottom && scrollTop + clientHeight >= scrollHeight) {
+      setIsScrolledToBottom(true)
```

Comment:
-1 I don't think this is the best solution. It also results in the scrollbar changing once the end is reached. All we want to know is whether or not the `ReactVirtualized__Grid__innerScrollContainer` is higher than the `ReactVirtualized__Grid`. Since we also only want to show it once the full list was loaded, I think it's best to:
* add 1 to the rowCount
* return "That's all folks" component in `cellRenderer`
  ```
    if (!hasNext && rowCount === rowIndex) {
      return columnIndex === 0 && <CellMeasurer >... That's all folks ...
  ```
  with `gridColumn: 1 / span ${columnCount}`

Reply:
You are right! I've changed the approach in #6338